batch.py

```diff
@@ -14,13 +14,15 @@
     parser = argparse.ArgumentParser(description='Run flu builds on aws')
     parser.add_argument('-l', '--lineages', nargs='+', type = str,  help ="flu lineages to include")
     parser.add_argument('-r', '--resolutions', nargs='+', type = str,  help ="flu resolutions to include")
+    parser.add_argument('-s', '--segments', nargs='+', type = str, help ="flu segments to include")
```

Comment:
If you include `defaults = ["ha", "na"]` here, you can remove the check for `params.segments is None` below. This will also expose the default segments to the user on the command line, in the future when the ArgumentParser's formatter class is set to `argparse.ArgumentDefaultsHelpFormatter`.

Reply:
Thanks for the suggestion @huddlej. I did it this way to follow the standard that already existed for `lineages` and `resolutions`. Changing now for all three.

Reply:
Good call.